### PR TITLE
OpenVino Ops

### PR DESCRIPTION
**Pull request by cnamrata15**
### Ops
Equal
Less Equal
Less
Not Equal
Maximum
Minimum

**Note**
Tests on the comparison Ops fail with the 
Error " C++ exception with description "Output 0 type 'f32' does not match Result type 'boolean'" thrown in the test body."
This error is on the ngraph interpreter side.

### CODE REVIEW

**Comment by tzerrell** [comment]
Are maximum and minimum passing tests?

File: plaidml/bridge/openvino/src/plaidml_plugin/ops/maximum.cpp [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+#include "plaidml/op/op.h"
+
+using namespace plaidml;          // NOLINT[build/namespaces]
+using namespace InferenceEngine;  // NOLINT[build/namespaces]
+
+namespace PlaidMLPlugin {
+
+static OpRegistration reg("maximum", [](const Context& ctx) {
+  // Note that nGraph ReLU does not include an alpha parameter
```
Comment (tzerrell) [inline plaidml/bridge/openvino/src/plaidml_plugin/ops/maximum.cpp:18]: Inapplicable comment

File: plaidml/bridge/openvino/src/plaidml_plugin/ops/minimum.cpp [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+#include "plaidml/op/op.h"
+
+using namespace plaidml;          // NOLINT[build/namespaces]
+using namespace InferenceEngine;  // NOLINT[build/namespaces]
+
+namespace PlaidMLPlugin {
+
+static OpRegistration reg("minimum", [](const Context& ctx) {
+  // Note that nGraph ReLU does not include an alpha parameter
```
Comment (tzerrell) [inline plaidml/bridge/openvino/src/plaidml_plugin/ops/minimum.cpp:18]: Inapplicable comment

**Comment by cnamrata15** [comment]
> Are maximum and minimum passing tests?

I think so. Will cross check to be sure.

**Pull request merged**